app/models/components/course/groups_ability_component.rb

```diff
@@ -22,6 +22,7 @@ def allow_group_manager_manage_group
   end
 
   def course_group_manager_hash
-    { group_users: { user_id: user.id, role: Course::GroupUser.roles[:manager] } }
+    { group_users: { course_user_id: user.course_user_ids,
```

Comment:
Yup, apologies, the has I used was singular. The problem is that the query didn't reference the other tables. Will look into this more after I fix the other PRs.

```ruby
ActiveRecord::StatementInvalid:
       PG::UndefinedTable: ERROR:  missing FROM-clause entry for table "course_user"
       LINE 1: ..." WHERE "course_group_users"."group_id" = $1 AND ("course_us...
                                                                    ^
       : SELECT COUNT(*) FROM "course_group_users" WHERE "course_group_users"."group_id" = $1 AND ("course_user"."user_id" = 55 AND "course_group_users"."role" = 1)
     # ./spec/models/course/group_ability_spec.rb:29:in `block (4 levels) in <top (required)>'
     # ./spec/support/bullet.rb:7:in `block in extended'
     # ------------------
     # --- Caused by: ---
     # PG::UndefinedTable:
     #   ERROR:  missing FROM-clause entry for table "course_user"
     #   LINE 1: ..." WHERE "course_group_users"."group_id" = $1 AND ("course_us...
     #                                                                ^
     #   ./spec/models/course/group_ability_spec.rb:29:in `block (4 levels) in <top (required)>'
```